src/overlays/ovl_i4/fox_ka.c

```diff
@@ -1,63 +1,2183 @@
+            D_801779C0 = gActors[4].obj.pos.z;
+            gActors[4].obj.rot.z -= 0.2f;
+            gActors[5].obj.rot.z += 0.3f;
+            gActors[6].obj.rot.z += 0.7f;
+            if (gCsFrameCount == 100) {
+                gCsFrameCount = 80;
+                player->unk_1D0 = 12;
+            }
+            break;
+
+        case 12:
+            gActors[4].vel.y += 0.3f;
+            gActors[4].vel.z += 0.7f;
+            gActors[4].obj.rot.z -= 1.0f;
+            if (gCsFrameCount == 120) {
+                func_8002F180();
+                player->unk_1D0 = 13;
+                player->pos.x = 0.0f;
+                player->pos.y = 1300.0f;
+                player->pos.z = 10000.0f;
+                player->unk_0E4 = -10.0f;
+                for (i = 1, actor = &gActors[1]; i < 4; i++, actor++) {
+                    actor->obj.pos.x = D_i4_8019F168[i - 1].x + player->pos.x;
+                    actor->obj.pos.y = D_i4_8019F168[i - 1].y + player->pos.y;
+                    actor->obj.pos.z = D_i4_8019F168[i - 1].z + player->pos.z;
+                    actor->unk_0F4.y = D_i4_8019F18C[i - 1];
+                    actor->unk_0F4.x = -10.0f;
+                    actor->state = 1;
+                    actor->timer_0BC = 1000;
+                }
+            }
+            break;
+
+        case 13:
+            player->camAt.x = D_801779A0 = player->pos.x;
+            player->camAt.y = D_801779B8 = player->pos.y;
+            player->camAt.z = D_801779C0 = player->pos.z;
+            player->camEye.x = D_80177978 = 100.0f;
+            player->camEye.z = D_80177988 = 7000.0f;
+            if (gCsFrameCount == 240) {
+                Object_Kill(&gActors[4].obj, gActors[4].sfxPos);
+                Object_Kill(&gActors[6].obj, gActors[6].sfxPos);
+                player->state_1C8 = PLAYERSTATE_1C8_3;
+                player->unk_014 = 0.0001f;
+                func_8001D444(0, D_80177C90, 0, 255);
+                D_80177838 = 80;
+                for (actor = &gActors[1], i = 1; i < 4; i += 1, actor++) {
+                    actor->timer_0BC = 0;
+                }
+                D_8015F928 = -610;
+            }
+            break;
+    }
+    src.x = 0.0f;
+    src.y = 0.0f;
+    src.z = 100.0f;
+    switch (gCsFrameCount) {
+        case 15:
+            Radio_PlayMessage(gMsg_ID_18000, RCID_BILL);
+            break;
+
+        case 10:
+        case 20:
+        case 25:
+        case 40:
+            actor = &gActors[5];
+            Matrix_RotateY(gCalcMatrix, actor->obj.rot.y * M_DTOR, 0);
+            Matrix_RotateX(gCalcMatrix, actor->obj.rot.x * M_DTOR, 1);
+            Matrix_MultVec3fNoTranslate(gCalcMatrix, &src, &dest);
+            func_8006EEFC(5, actor->obj.pos.x + (dest.x * 1.5), actor->obj.pos.y + (dest.y * 1.5),
+                          actor->obj.pos.z + (dest.z * 1.5), dest.x, dest.y, dest.z, actor->obj.rot.x, actor->obj.rot.y,
+                          actor->obj.rot.z);
+            break;
```

Comment:
Can the actor temp be removed from this case?

Reply:
Sadly nope